doc/languages-frameworks/chicken.section.md

```diff
@@ -0,0 +1,42 @@
+# CHICKEN {#sec-chicken}
+
+[CHICKEN](https://call-cc.org/) is a
+[R⁵RS](https://schemers.org/Documents/Standards/R5RS/HTML/)-compliant Scheme
+compiler. It includes an interactive mode and a custom package format, "eggs".
+
+## Using Eggs
+
+Eggs described in nixpkgs are available inside the
+`chickenPackages.chickenEggs` attrset. Including an egg as a build input is
+done in the typical Nix fashion. For example, to include support for [SRFI
+189](https://srfi.schemers.org/srfi-189/srfi-189.html) in a derivation, one
+might write:
+
+```nix
+  buildInputs = [
+    chicken
+    chickenPackages.chickenEggs.srfi-189
+  ];
+```
+
+## Updating Eggs
+
+nixpkgs does not know about all eggs, but includes support for
+[egg2nix](https://github.com/the-kenny/egg2nix), which can be used to
+automatically include published eggs.
+
+The procedure consists of running the following shell commands:
```

Comment:
Does this only regenerate the package set? Or does it also update the versions involved? Are the versions pinned?

```suggestion
The package set is regenerated by running the following shell commands:
```

Reply:
Versions are pinned. We take an all-or-nothing snapshot of working packages, akin to our Node.js integration. This is probably what we desire, but it does mean that introducing new eggs must happen after we update the existing eggs. I'll add this to the doc, too.